src/components/Footer/FooterInfo/FooterInfo.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+import FooterLink from '../FooterLink/FooterLink';
+
+const FOOTER_INFO_DATA = [
+  { title: '소개', infoList: ['회사소개', '채용', '공고'] },
+  {
+    title: '파트너',
+    infoList: ['파트너 등록하기', 'Affiliate 프로그램', '리얼파트너'],
+  },
+  { title: '지원', infoList: ['자주 묻는 질문', '최저가 보장제'] },
```

Comment:
```suggestion
  { id:1, title: '소개', infoList: ['회사소개', '채용', '공고'] },
  {id:2,
    title: '파트너',
    infoList: ['파트너 등록하기', 'Affiliate 프로그램', '리얼파트너'],
  },
  { id:3, title: '지원', infoList: ['자주 묻는 질문', '최저가 보장제'] },
```
* 객체마다 id를 적어줘서 
* map에서 key를 객체별 id로 적용하는 방식이 
* index로 쓰는방식보단 긍정적입니다.
* 지금은 index로 쓸 순 있지만, index로 쓰는건 최최최최최종의 방법으로 고려 해 주세요
* [index를 key로 쓸 경우 부정적인 영향에 대한 자료는 여기 있습니다](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318)

Reply:
반영하였습니다.